dist/settings/settings.html

```diff
@@ -3,4 +3,4 @@
mages/play_gl_6x8.63df6e88.png"> on a song to play the rest of the album</li> </ul> <h3 class="title">Keyboard Shortcuts</h3> <ul class="preferences"> <li><em>Esc</em> Hide Queue</li> <li><em>[1-6]</em> Switch to corresponding tab</li> <li><em>[a-z]</em> Use to Quickly Browse to an Artist</li> <li><em>Home</em> Scroll to Top of Artist List</li> <li><em>Spacebar</em> Play/Pause</li> <li><em>&rarr;</em> Next Track</li> <li><em>&larr;</em> Previous Track</li> <li><em>-/_</em> Volume Down <em>=/+</em> Volume Up</li> <li><em>Media Keys</em> via <a href="https://chrome.google.com/webstore/detail/key-socket-media-keys/fphfgdknbpakeedbaenojjdcdoajihik" target="_blank">Key Socket Media Keys</a> or <a href="https://github.com/carlin-q-scott/browser-media-keys/pull/21">browser-media-keys</a></li> </ul> <div id="donate" class="subsection floatleft"> <h3 class="title">Buy me a <span class="beer">beer</span>! I'd like that :)</h3> <form id="paypal" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank"> <input type="hidden" name="cmd" value="_s-xclick"> <input type="hidden" name="hosted_button_id" value="VMTENRSJWQ234"> <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" alt="PayPal - The safer, easier way to pay online!"> <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1"> </form> <small>*This is a donation to <a href="https://twitter.com/JamstashApp" target="_blank">tsquillario</a>, the developer of <a href="https://github.com/tsquillario/Jamstash" target="_blank">Jamstash</a>. <br>Not related to a Subsonic License!</small> </div> </div> <div class="subsection floatleft"> <h3 class="title">Links</h3> <ul class="preferences"> <li>Report Issues - <a href="https://github.com/tsquillario/Jamstash" target="_blank">https://github.com/tsquillario/Jamstash</a></li> <li>Jamstash Chrome App - <a href="https://chrome.google.com/webstore/detail/jccdpflnecheidefpofmlblgebobbloc" target="_blank">Chrome Web Store</a></li> <li>Beta Site - <a href="http://beta.jamstash.com" target="_blank">http://beta.jamstash.com</a></li> <li><a href="https://twitter.com/JamstashApp" target="_blank">Follow @JamstashApp</a> </li></ul> <h3 class="title">Thanks</h3> <ul class="preferences"> <li>Icons - <a href="http://somerandomdude.com/work/iconic" target="_blank">http://somerandomdude.com/work/iconic</a></li> <li>Audio Library - <a href="http://jplayer.org" target="_blank">http://jplayer.org</a></li> </ul> </div> <div class="clear"></div> <div class="subsection floatleft"> <h3 class="title">Change Log</h3> <ul id="ChangeLog" class="preferences"> <li class="log" ng-repeat="o in changeLog"><span class="version">{{o.date + ' - ' + o.version}}</span> <span class="changes" ng-repeat="a in o.changes" ng-bind-html="a.text"></span> </li> </ul> <a href="" ng-click="changeLogShowMore()">Show More</a> </div> <div class="clear"></div> </div> <div class="clear"></div> </div>
```

Comment:
Please change `/app/settings/settings.html`, run `grunt` and then commit.

Files in the `dist/` folder are not meant to be edited, as you can see this is a "minified" HTML file to reduce file transfer time.